datadog_checks_base/datadog_checks/base/utils/db/utils.py

```diff
@@ -183,6 +183,27 @@ def resolve_db_host(db_host):
     return db_host
 
 
+def get_agent_host_tags():
+    '''
+    Get the tags from the agent host and return them as a list of strings.
+    '''
+    result = []
+    host_tags = datadog_agent.get_host_tags()
+    if not host_tags:
+        return result
+    try:
+        tags_dict = json.loads(host_tags) or {}
+    except:
```

Comment:
nit: `except` will likely throw linter errors, i think you need `except Exception as e`